blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -344,42 +344,18 @@ export class WorkspaceSearch {
     if (!this.blocks_.length) {
       return;
     }
-    this.clearCurrentBlock_();
+    if (this.currentBlock_) {
+      this.unhighlightCurrentSelection_(this.currentBlock_);
+    }
     this.currentBlockIndex_ =
         (index % this.blocks_.length + this.blocks_.length) %
         this.blocks_.length;
     this.currentBlock_ = this.blocks_[this.currentBlockIndex_];
-    const path = this.currentBlock_.pathObject.svgPath;
-    Blockly.utils.dom.addClass(path, 'search-current');
-    this.updateCursor_(this.currentBlock_);
+    this.highlightCurrentSelection_(this.currentBlock_);
+    this.updateCursor_();
```

Comment:
Why are we not passing in the block here?

Reply:
I've changed to back to accepting block as param as it seems more consistent